packages/marketplace/src/reducers/__tests__/developer-detail.ts

```diff
@@ -0,0 +1,57 @@
+import developerDetailReducer, { defaultState } from '../developer-detail'
```

Comment:
This one also has fetchMyIdentity Action, pls reuse that instead of writing the new things